### PR TITLE
db migration

### PR DESCRIPTION
**Pull request by cecobask**

### CODE REVIEW

File: docs/intro/config.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ Core of the Graphback will take your GraphQL Schema types annotated by `model` a
...
 
-Graphback allows the user to customize all process by using open configuration stardard `graphql-config`:
+Graphback allows the user to customize all process by using open configuration standard `graphql-config`:
 https://graphql-config.com/docs/introduction
 
 ## Configuration
 
-Each project needs to have graphqlrc.yml file available in the root folder. 
-This file is also automatically generated when using `graphback config` command
+Each project needs to have .graphqlrc.yml file available in the root folder.
```
Comment (craicoverflow) [inline docs/intro/config.md]:
```suggestion
Each project needs to have `.graphqlrc.yml` file available in the root folder. 
```

File: packages/graphback-cli/src/components/db.ts [hunk 2]
```diff
@@ -13,57 +12,49 @@ const handleError = (err: { code: string; message: string; }): void => {
...
+    if (!dbMigrationConfig) {
+      throw new Error(`You should provide a valid '${dbmigrationsExtension}' config to migrate schema`);
     }
 
-    if (!graphbackConfig.model) {
-      throw new Error(`'${extensionName}' config missing 'model' value that is required`);
-    }
-
-    const models = new GlobSync(`${graphbackConfig.model}/*.graphql`)
+    const models = new GlobSync(project.schema as string);
```
Comment (craicoverflow) [inline packages/graphback-cli/src/components/db.ts]: You could define `const schema = await project.getSchema("string");` before this and use it here.

File: packages/graphback-cli/src/components/db.ts [hunk 3]
```diff
@@ -13,57 +12,49 @@ const handleError = (err: { code: string; message: string; }): void => {
...
 
-    if (!graphbackConfig.dbmigrations) {
-      throw new Error(`'${extensionName}' config missing 'dbmigrations' section that that is required. Please check example application for more information`);
-    }
-
-    if (graphbackConfig.dbmigrations.client === 'sqlite3') {
-      logInfo(`SQLLite database will need to be recreated from scratch with every migration. Please manually remove/backup existing SQLLite database file before migration`)
-      await execa('touch', [graphbackConfig.dbmigrations.connection.filename])
+    if (dbMigrationConfig.client === 'sqlite3') {
+      logInfo(`SQLLite database will need to be recreated from scratch with every migration. Please manually remove/backup existing SQLLite database file before migration`);
```
Comment (craicoverflow) [inline packages/graphback-cli/src/components/db.ts]:
```suggestion
      logInfo(`SQLite database will need to be recreated from scratch with every migration. Please manually remove/backup existing SQLLite database file before migration`);
```

**Comment by wtrocki** [comment]
Closing as this PR is an duplicate

**Pull request closed without merging**